splunk_eventgen/splunk_app/README/eventgen.conf.spec

```diff
@@ -22,7 +22,7 @@
 [global]
 disabled = false
 debug = false
-verbose = false
+verbosity = false
```

Comment:
Make the spec consistent for this change: https://github.com/splunk/eventgen/pull/191